l10n_br_fiscal/models/document_fiscal_line_mixin_methods.py

```diff
@@ -394,10 +403,13 @@ def _onchange_fiscal_operation_line_id(self):
 
             self.ipi_guideline_id = mapping_result['ipi_guideline']
             self.cfop_id = mapping_result['cfop']
-            taxes = self.env['l10n_br_fiscal.tax']
-            for tax in mapping_result['taxes'].values():
-                taxes |= tax
-            self.fiscal_tax_ids = [(6, 0, taxes.ids)]
+
+            if mapping_result.get('taxes'):
+                taxes = self.env['l10n_br_fiscal.tax']
+                for tax in mapping_result['taxes'].values():
+                    taxes |= tax
+                self.fiscal_tax_ids = [(6, 0, taxes.ids)]
+
             self._update_taxes()
             self.comment_ids = self.fiscal_operation_line_id.comment_ids
```

Comment:
Se o método de calculo for manual não vai copiar as observações da linha da operação fiscal